src/Gui/NaviCube.cpp

```diff
@@ -1014,6 +1014,7 @@ SbRotation NaviCubeImplementation::getNearestOrientation(PickId pickId) {
     static const float pi2 = boost::math::constants::two_pi<float>();
     static const float pi1_2 = boost::math::constants::half_pi<float>();
     static const float pi1_3 = boost::math::constants::third_pi<float>();
+    static const float pi1_4 = boost::math::constants::quarter_pi<float>();
```

Comment:
These boost constants are only used in something like three files, we should get rid of them (either as part of this PR or as a follow-up).

Reply:
I was leaving these out as I tried to keep this PR small, but I'll fix it :)